### ISSUE TITLE
Many training epochs in each Monte Carlo step

### ISSUE THREAD

**Issue by VolodyaCO** [netket/netket#544]
I understand that currently netket provides one training epoch for each MC step (after the metropolis sampling). Is there any hyperparameter where I can tune how many training epochs are carried out after each MC step? Judging by the results in [this paper](https://journals.aps.org/prresearch/pdf/10.1103/PhysRevResearch.2.012039) it could lead to large improvements in convergence time.

**Comment by gcarleo** [comment]
that works with re-sampling, if I remember correctly, so you would also need to reweight the gradients etc with the correct factor... it could be implemented but I don't see a strong case for it, I think that sampling is not a big bottleneck in deep learning frameworks as it used to be when that paper was written
